src/test/java/ibm/jceplus/junit/base/BaseByteArrayOutputDelayTest.java

```diff
@@ -82,7 +82,7 @@ private static void testRandom(int iterations) throws IOException {
             byte[] testData = new byte[dataSize];
             random.nextBytes(testData);
 
-            int chopSize = random.nextInt(4096);
+            int chopSize = random.nextInt(4095) + 1; // Generate a random between 1 and 4096.
             int delayByte = random.nextInt(dataSize);
```

Comment:
Even if this is set to `0`, it won't fail. But it doesn't really test the `ByteArrayOutputDelay`, since it ends up just using the `ByteArrayOutputStream`. I think we should change that too.

Reply:
This particular test should randomly test, or not test delay, depending on if the value is 0 or not ( randomly ). We do have other tests in this file that explicitly test 0 for the delay. This seem like a reasonable approach ?

Reply:
Sure.